arcaflow_plugin_utilities/utilities_plugin.py

```diff
@@ -3,16 +3,25 @@
 import sys
 import uuid
 from datetime import datetime, timezone
+import time
 import typing
 from dataclasses import dataclass
 from arcaflow_plugin_sdk import plugin, schema
 
 
+# For steps that do not require input
 @dataclass
 class InputParams:
-    """
-    This is the data structure for the input parameters of the uuid plugin.
-    """
+    {}
```

Comment:
The `{}` doesn't look conventional to me.

If you really want a no-op (and don't want to let the comment stand in for that), I think the conventional token is `pass`.

Reply:
It seems that pylint suggests an empty class instead of `pass`.

Reply:
"Empty class" doesn't work (expects an indented line) ... however apparently the indented docstring works as well as an explicit `pass` for the parser. Huh.

Reply:
Yes...there has to be _something_ after the colon -- any expression (even `9` will do) -- so now we're down to debating what that should be.  `pass` is the canonical way to express a "no-op".  The ellipsis (`...`) is available.  `None` might be OK.  (An empty dict literal is way down my personal list....)

However, if you adhere to the practice that all classes should have docstrings, then the docstring itself meets the requirement (after all, it's just an expression), and we don't have to debate what to put there.